docs/examples/manifests/user_monitoring/2_configure_user_workload_monitoring.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
```

Comment:
it should be like this?

Reply:
Yes, it's an example which is "default" of course there can be some extra config ( pointer is ) in the documentation, here we want default